platform-hub-api/spec/controllers/concerns/authentication_spec.rb

```diff
@@ -35,14 +35,28 @@ def index
       test_auth_token
     end
 
-    it 'should check for a current user and create one' do
-      user = instance_double('User')
+    context 'when user is active' do
+      let(:user) { build(:user) }
```

Comment:
Is `is_active` set to `true` by default in the factory? I know the table column is defaulted to `true`, but it may be worth explicitly setting this in the factory too? (though very minor!)